tests/e2e/specs/free-blocks.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Internal dependencies
+ */
+import BlockEditorPage from '../lib/pages/wp-admin/block-editor';
+import PostFrontendPage from '../lib/pages/postFrontend';
+import { connectThroughWPAdminIfNeeded } from '../lib/flows/jetpack-connect';
+import { resetWordpressInstall, getNgrokSiteUrl } from '../lib/utils-helper';
+import PinterestBlock from '../lib/blocks/pinterest';
+import EventbriteBlock from '../lib/blocks/eventbrite';
+
+describe( 'Free blocks', () => {
+	beforeAll( async () => {
+		await resetWordpressInstall();
+		const url = getNgrokSiteUrl();
+		console.log( 'NEW SITE URL: ' + url );
+
+		await connectThroughWPAdminIfNeeded( { mockPlanData: true } );
```

Comment:
@scruffian for some reason the tests fail on CI if  mockPlanData is false with:
```
TimeoutError: Timeout exceeded while waiting for event
      93 | 		await jetpackPage.reload( { waitFor: 'networkidle0' } );
      94 | 
    > 95 | 		await page.waitForResponse(
         | 		           ^
      96 | 			response => response.url().match( /v4\/site[^\/]/ ) && response.status() === 200,
      97 | 			{ timeout: 60 * 1000 }
      98 | 		);
```
but, haven't managed to work out why yet, and looking through the code it appears that setting it to true loads a jetpack business plan, which isn't want we want to test free blocks, will do some more digging update if I work out what is going on here.

Reply:
Interesting. That code was added to be sure that response with plan data come in before moving forward. It might be flaky, and at this point, it is not used at all. 

As an option, the conditional could be updated to `if ( ! mockPlanData && plan !== 'free' ) {`

Reply:
updating the conditional as suggested seems to have fixed this for now.